hedera-mirror-rest/__tests__/globalSetup.js

```diff
@@ -0,0 +1,89 @@
+/*-
+ * ‌
+ * Hedera Mirror Node
+ * ​
+ * Copyright (C) 2019 - 2022 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import crypto from 'crypto';
+import pg from 'pg';
+import {GenericContainer} from 'testcontainers';
+
+const DEFAULT_DB_NAME = 'mirror_node_integration';
+const POSTGRES_PORT = 5432;
+
+const v1DockerImageConfig = {
+  imageName: 'postgres',
+  tagName: '14-alpine',
+};
+
+const v2DockerImageConfig = {
+  imageName: 'citusdata/citus',
+  tagName: '10.2.2-alpine',
+};
+
```

Comment:
I wonder why we separate these two here only to re-combine them when we use them. Would be simpler to just have:
```suggestion
const v1DatabaseImage = 'postgres:14-alpine';
const v2DatabaseImage = 'citusdata/citus:11-alpine';
```

Reply:
probably for the convenience of only having to update the tag.

will make the suggested changed